stumpui/windows.lisp

```diff
@@ -0,0 +1,215 @@
+;;;; Copyright (C) 2016 Joram Schrijver
```

Comment:
What happened to window.lisp? Can we find a name that represents the content of this file better? window-protocol.lisp maybe?